adapters/bridge.go

```diff
@@ -127,3 +128,9 @@ func (bp bridgeOutgoing) MarshalJSON() ([]byte, error) {
 	}
 	return json.Marshal(anon)
 }
+
+var zeroURL *url.URL
+
+func init() {
```

Comment:
Do we need the init function? Can we do this in the same line in the variable declaration like we do [here](https://github.com/smartcontractkit/chainlink/blob/master/services/subscription.go#L42)?